### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by nbelakovski** [nbelakovski/topography_neural_net#2]
-Rearrange neural net training loop into epochs over entire training set, and a loop over the mini-batches.
-Shuffle the training set on each epoch.
-Set up the network to pick up from a checkpoint.
-Clean up comments and extraneous things in the network file.
-Optional: use tensorflow serving tools instead of the semi-hacked inference.py

**Comment by nbelakovski** [comment]
All done, except the last one. New issue can be opened for that.
